docs/operator-guides/operator-installation.md

```diff
@@ -65,6 +65,37 @@ curl -sSfL https://raw.githubusercontent.com/layr-labs/eigenlayer-cli/master/scr
 
 ### Install CLI using Go
 
+Firstly, let's make sure your system has the latest package information.
```

Comment:
few points
- This is very ubuntu distro specific instruction
- Also adding a go version will get obsolete over time
- a user can have GO PATH in custom location

I think we should not add it, instead maybe link to the Go website installation doc - https://go.dev/doc/install 
- We don't need to worry about version and any updates
- It comes up with all OSes suggestions